tests/test/url-with-defaults.test.js

```diff
@@ -0,0 +1,15 @@
+delete process.env.UNLOCK_HOST
```

Comment:
Is that going to have an effect only on this file?

Reply:
I guess we will find out, but since each test runs in its own process, I assumed yes.

Reply:
the answer is it only affects this file